lib/js_routes.rb

```diff
@@ -36,6 +36,7 @@ class JsRoutes
 
   LAST_OPTIONS_KEY = "options".freeze
   FILTERED_DEFAULT_PARTS = [:controller, :action, :subdomain]
+  URL_OPTIONS = [:protocol, :domain, :host, :port]
```

Comment:
None of the tests is failing when subdomain exists in `URL_OPTIONS`.
I've added a subdomain to `URL_OPTIONS`.